rust/src/datatypes.rs

```diff
@@ -420,8 +412,8 @@ impl TryFrom<&ArrowField> for Field {
             name: field.name().clone(),
             logical_type: LogicalType::try_from(field.data_type())?,
             encoding: match field.data_type() {
-                dt if is_numeric(dt) => Some(Encoding::Plain),
-                dt if is_binary(dt) => Some(Encoding::VarBinary),
+                dt if dt.is_numeric() || matches!(dt, DataType::Boolean) => Some(Encoding::Plain),
```

Comment:
maybe move `is_fixed_stride` from datatypes to arrow.rs and use that here?

Reply:
done.